apps/miniapp/domains/condo/schema/User.js

```diff
@@ -0,0 +1,100 @@
+/**
+ * Generated by `createschema condo.User 'name:Text;isAdmin:Checkbox;isSupport:Checkbox;type:Text' --force`
+ */
+
+const get = require('lodash/get')
+
+const { Text, Select, Integer, Checkbox, Password } = require('@keystonejs/fields')
+const { GQLListSchema } = require('@core/keystone/schema')
+const { uuided, tracked, softDeleted } = require('@core/keystone/plugins')
+
+// TODO(pahaz): move it to core? or vendor this plugin?
+const { dvAndSender } = require('@condo/domains/common/schema/plugins/dvAndSender')
+
+const access = require('@miniapp/domains/condo/access/User')
+const { SERVICE_USER_TYPE, USER_TYPES } = require('@miniapp/domains/condo/constants/user')
+
+const User = new GQLListSchema('User', {
+    schemaDoc: 'condo.User (exported from condo API)',
+    fields: {
+        v: {
+            schemaDoc: 'condo.User.v',
+            type: Integer,
+            isRequired: true,
+            defaultValue: 1,
+            access: access.canAccessToIsAdminField,
+            hooks: {
+                resolveInput: ({ resolvedData, existingItem }) => {
+                    if (get(existingItem, 'isLocal') || get(resolvedData, 'isLocal')) {
+                        if (existingItem) {
+                            resolvedData.v = Number(existingItem.v || 1) + 1
+                        }
+                    }
+                    return resolvedData.v
+                },
+            },
+        },
+        name: {
+            schemaDoc: 'condo.User.name',
+            type: Text,
+            access: access.canAccessToIsAdminField,
+        },
+        type: {
+            schemaDoc: 'condo.User.type',
+            type: Select,
+            dataType: 'enum',
+            options: USER_TYPES,
+            defaultValue: SERVICE_USER_TYPE,
+            isRequired: true,
+            access: access.canAccessToIsAdminField,
```

Comment:
The name `access.canAccessToIsAdminField ` confuses in contexts of other fields.
Could you please change the name?